src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift

```diff
@@ -250,6 +250,7 @@ class ENAUITests_04a_ExposureSubmission: CWATestCase {
 
 	}
 
+	/* Commented out in favour of SRS story
```

Comment:
same for the following ...

Reply:
It is commented out because at the moment we don't have the TAN